packages/vc-handler-ld-local/src/types/types.ts

```diff
@@ -218,14 +218,25 @@ export interface IVerifyPresentationLDArgs {
   /**
    * Use this presentation purpose for the verification method in the DID when doing a check (defaualts to assertionMethod)
    */
-  presentationPurpose?: typeof ProofPurpose
+  presentationPurpose?: IProofPurpose
 
   /**
    * Check status function, to check credentials that have a credentialStatus property
    */
   checkStatus?: Function
 }
 
+export interface IProofPurpose {
```

Comment:
I could have made different types and or them like AuthenticationProofPurpose|  ControllerProofPurpose | AssertionProofPurpose, but I'm not so confident the API extractor will turn this into a correct OpenAPI v3 structure.

Reply:
Please do try that first though. The Proof Purposes are well known constructs in any SSI solution. This is really confusing to people that know SSI.